services/pane.js

```diff
@@ -156,17 +181,31 @@ function addPreview(preview) {
   }
 }
 
+/**
+ * format and assemble error messages
+ * @param {Object[]} errors
+ * @returns {Element}
+ */
 function addErrors(errors) {
   return _.reduce(errors, function (el, error) {
-    var errorEl = dom.create(`
-        <div class="publish-error">
-          <span class="label">${error.rule.label}:</span>
-          <span class="description">${error.rule.description}</span>
-          <ul class="errors"></ul>
-        </div>
-      `),
+    var errorEl = dom.find('.publish-errors'),
+      errorLabel = dom.find(errorEl, '.label'),
+      errorDescription = dom.find(errorEl, '.description'),
       list = dom.find(errorEl, 'ul');
 
+    // reset template label default if available
```

Comment:
For readability, the template skeleton in the nunjucks file has a value. Also, if there is a problem with populating the error, but we still have an error condition where this dialog pane would be displayed, the error would return something to the user instead of being blank.